packages/types/src/events/channel-membership.ts

```diff
@@ -0,0 +1,18 @@
+export interface MemberJoinedChannelEvent {
```

Comment:
Should this file be member.ts for consistency with most of the other file names? They seem to be derived from the first unique word/wordcombination all the contained types share in common?

Reply:
Sounds good to me! Will update shortly.